src/client.rs

```diff
@@ -84,6 +85,23 @@ impl ClientBuilder {
         })
     }
 
+    /// Disable proxy setting.
+    pub fn no_proxy(self) -> ClientBuilder {
+        self.with_inner(move |inner| inner.no_proxy())
+    }
+
+    /// Enable system proxy setting.
+    pub fn use_sys_proxy(self) -> ClientBuilder {
```

Comment:
Should this method also exist on the async builder? And here it can just call through, like you did with `no_proxy`?